src/components/book.js

```diff
@@ -14,11 +14,19 @@ const Book = (props) => {
 };
 
 Book.defaultProps = {
-  bookData: null,
+  bookData: {
```

Comment:
- [ ] Kindly display the author on rendered books.

Reply:
Done!